### PR TITLE
style: Admin page grid view and icons for github, video, telegram

### PR DESCRIPTION
**Pull request by ChangoMan**
## Description

Adjusting the Admin page cards to fit in a grid view and move around some of the items within the card. Also adding icons for github, video link, and telegram. More admin fixes will be incoming, this is just an initial iteration.

### Screenshots

Screenshot 2024-08-23 at 12 11 00 PM

### CODE REVIEW

File: packages/nextjs/app/admin/_components/SubmissionCard.tsx [hunk 1]
```diff
@@ -87,89 +87,126 @@ export const SubmissionCard = ({ submission }: { submission: Submission }) => {
...
+                  stroke="currentColor"
+                  strokeWidth="2"
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                >
+                  <path d="M15 22v-4a4.8 4.8 0 0 0-1-3.5c3 0 6-2 6-5.5.08-1.25-.27-2.48-1-3.5.28-1.15.28-2.35 0-3.5 0 0-1 0-3 1.5-2.64-.5-5.36-.5-8 0C6 2 5 2 5 2c-.3 1.15-.3 2.35 0 3.5A5.403 5.403 0 0 0 4 9c0 3.5 3 5.5 6 5.5-.39.49-.68 1.05-.85 1.65-.17.6-.22 1.23-.15 1.85v4" />
+                  <path d="M9 18c-4.51 2-5-2-7-2" />
+                </svg>
+              </a>
+            )}
```
Comment (carletex) [inline packages/nextjs/app/admin/_components/SubmissionCard.tsx:115]: Maybe we can add this (and the video one) as an SVG file as you did with the telegram one? Or even a tsx file if we need to have props on it.

**Comment by carletex** [comment]
This is great @ChangoMan 

just left a tiny review comment. After that, we can merge and keep iterating in another PR.

Thanks!

File: packages/nextjs/app/admin/_components/SubmissionCard.tsx (hunk 1 above)
Comment (ChangoMan) [reply packages/nextjs/app/admin/_components/SubmissionCard.tsx]: Got this moved to a file

File: packages/nextjs/.eslintrc.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "@typescript-eslint/no-unused-vars": ["error"],
     "@typescript-eslint/no-explicit-any": ["off"],
     "@typescript-eslint/ban-ts-comment": ["off"],
+    "@next/next/no-img-element": ["off"],
```
Comment (ChangoMan) [inline packages/nextjs/.eslintrc.json:8]: Was getting a linting error because of this rule

Comment (carletex) [reply packages/nextjs/.eslintrc.json:8]: Yeah, we have that so people use NextJS <Image /> which comes with [some optimizations](https://nextjs.org/docs/app/building-your-application/optimizing/images)

```
<Image alt="github icon" width="24" height="24" src="/icon-github.svg" />
```

Not a big deal, especially for these tiny images

**Review by carletex — APPROVED** [review]

**Pull request merged**